solara/server/server.py

```diff
@@ -452,8 +452,9 @@ def hash_extension(name):
                 file_path = directory / (name + ".js")
                 if file_path.exists():
                     for file in directory.glob("**/*.*"):
-                        data = file.read_bytes()
-                        h.update(data)
+                        if file.is_file():  # Skip directories
```

Comment:
```suggestion
                        if file.is_file():  # Otherwise directories with a dot in the name are included
```

I think it would be a good idea to indicate in the comment why we need to explicitly skip directories, since one might expect that the above pattern doesn't include any.